Plots/Contours/NCL_lb_2.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
When I try to run NCL_lb_2.py I get:

Traceback (most recent call last):
  File "NCL_lb_2.py", line 45, in <module>
    cbar = fig.colorbar(a, label='', ticks = ticks, shrink=0.4)
NameError: name 'a' is not defined